arm-software/linux/build.sh

```diff
@@ -190,6 +191,8 @@ Environment Variables:
                         (default: ${MKMODULEDIRS_PATH})
     SOURCES_DIR         The directory where all source code will be stored
                         (default: $SOURCES_DIR)
+    BOLTTESTS_DIR       The optionsl directory where the bolt-tests repo has been cloned
```

Comment:
nit:
```suggestion
    BOLTTESTS_DIR       The options directory where the bolt-tests repo has been cloned
```

Reply:
Thanks for spotting this, it meant to be `optional`.